applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -331,6 +336,9 @@ static int lc3_qos_cb(struct bt_audio_stream *stream, const struct bt_codec_qos
 	LOG_DBG("QoS: stream %p qos %p", (void *)stream, (void *)qos);
 	LOG_INF("Presentation delay %d us is set by initiator", qos->pd);
 	ret = audio_datapath_pres_delay_us_set(qos->pd);
+	if (ret) {
+		LOG_WRN("Failed to set QoS for stream %p", stream);
```

Comment:
This is not the QoS, but it's setting the pres delay in the audio system. This is going to change in https://github.com/nrfconnect/sdk-nrf/pull/10562 so I suggest just removing this if (ret) to reduce merge conflicts